pyresample/area_config.py

```diff
@@ -858,3 +859,34 @@ def convert_def_to_yaml(def_area_file, yaml_area_file):
     with open(yaml_area_file, 'w') as yaml_file:
         for area in areas:
             yaml_file.write(area.create_areas_def())
+
+
+def generate_area_def_rst_list(area_file):
+    """Create rst list of available area definitions with overview plot.
+
+    Args:
+        area_file (str): Path to area yaml file.
+
+    Returns:
+        str: rst list formatted string.
```

Comment:
For future reference I think I'd like pyresample to migrate to type annotations and no type information in the docstring.